nautobot/docs/development/index.md

```diff
@@ -93,17 +110,54 @@ Should a patch release contain a fix for security vulnerability(s) (i.e. CVE(s))
+For the abundance of clarity sake, we are officially naming what is sometimes called an "LTS" release of software a "Maintenance" or "Long Term Maintenance (LTM)" release of Nautobot. The mindset is we always aim to release stable software and new users choosing to install an LTS release because it appears to be more stable, will almost immediately be presented with breaking changes upon their next upgrade.
+
+Our LTM release will be the last minor of the previous major release. At the time of this writing `1.5.z` will be our maintenance release of Nautobot for the `1.y` release cycle. Version 1 of Nautobot had an extended release cycle as it was our first major release of the platform. Going forward it can be expected that the `x.3` version of the platform will be the maintenance train of any major version.
+
+With this schedule you can expect a few things per major/minor release:
+
+* `x.0.0`
+    * Introduce breaking changes
+    * Changing default behavior for APIs, functions
+* `x.1.z`
+    * Typical minor release, adding features, bug fixes, potential deprecations
+* `x.2.z`
+    * Removal of deprecated functions, classes introduced in the previous major release (ex: only if deprecated in `1.y`, removed in `2.2`).
+* `x.3.z`
+    * Maintenance release candidate.
+    * Features may be added in `x.3.0` but nothing further in this cycle.
+
+A release will only be marked as "In Maintenance" when the next major release is published. Active bug fixes will be applied to a `x.3` until that time. Once a new major release has been published, the following will apply to the `x.3` codebase:
+
+* Dependencies are frozen/pinned to a specific release. Will only be upgraded if addressing a vulnerability.
+* Data-loss and CVE related fixes will be back ported from the new active release cycle. All other fixes will be back ported on a case-by-case basis.
+* Patch releases for this phase will be on an as-needed basis.
+* Features from newer releases will NOT be back ported.
+
+A maintenance release will be actively maintained until the next maintenance release is available, roughly a year from the launch of the previous one.
+
+#### Launch of Maintenance Release Schedule (as an Example)
+
+At the time of this writing we are in the active development of Nautobot 1.5. This will be our last minor release of the v1 series of release and therefor become our first "Maintenance Release" of Nautobot. We will be actively apply the normal category of bug fixes (including UI tweaks, display bugs, etc.) until the release of Nautobot 2.0.
+
+Once we launch Nautobot 2.0, 1.5 will go into maintenance mode, continuing to receive data-loss and CVE related fixes. At that time we will encourage users to migrate to v2 as they are ready. Nautobot 1.5 will continue to receive these fixes until the release of Nautobot 2.3, where we will end the maintenance of Nautobot 1.5. Nautobot 2.3 will then become the maintenance release until Nautobot 3.3 is published the following year.
```

Comment:
```suggestion
Once we launch Nautobot 2.0, 1.5 will go into maintenance mode and be considered LTM, continuing to receive data loss and CVE-related fixes. At that time we will encourage users to migrate to v2 as they are ready. Nautobot 1.5 will continue to receive such fixes until the release of Nautobot 2.3, where we will end the maintenance of Nautobot 1.5 and it will formally become "End of Life" (EOL). Nautobot 2.3 will then become the LTM release until Nautobot 3.3 is published the following year.
```